platform/ext/target/nordic_nrf/common/nrfx_glue.h

```diff
@@ -156,7 +155,7 @@ void nrfx_critical_section_exit(void);
 //------------------------------------------------------------------------------
 
 /** @brief Atomic 32-bit unsigned type. */
-#define nrfx_atomic_t  nrfx_atomic_u32_t
```

Comment:
Can we just remove all the atomic defines in this file, instead of defining them to nothing?

Reply:
I've left this part unmodified compared to the template (https://github.com/NordicSemiconductor/nrfx/blob/master/templates/nrfx_glue.h#L156), as I wanted to avoid cutting out just parts of this file (atomic stuff could go away, but `NRFX_CUSTOM_ERROR_CODES` and `NRFX_EVENT_READBACK_ENABLED` defined later would need to stay).

Reply:
This way updates of nrfx should be easier when some changes in the `nrfx_glue.h` template appear.